### DISCUSSION TITLE
Lotes não encontrados

### DISCUSSION THREAD

**Discussion by maikrodrigues** [Chia-Network/chia-blockchain#4790] [General]
Meus plotes estavam todos normais, mas depois de 1 semana mais ou menos que iniciei na hpool meus plotes pararam de cultivar e está aparecendo lotes não encontrados. Já tirei tudo que eu tinha colocado na hpool, mas o problema ainda persisti.
Alguém poderia me ajudar?

**Comment by eltom933** [comment]
ptz o meu começou com isso hj.... já conseguiu resolver?

**Reply by maikrodrigues** [reply]
Consegui nada, se você encontrar uma solução me passa fazendo favor.
